samples/hosting/docker/Core_7/Receiver/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Build runtime image
+FROM microsoft/dotnet:runtime
```

Comment:
I'm wondering if one of the more specific tags might be better here: `2.0.0-runtime`, `2.0-runtime`, or `2-runtime` ?

Reply:
Done, I specified `2.0-runtime`.